### ISSUE TITLE
Animation breaks with more than 10 line traces with no specified color

### ISSUE THREAD

**Issue by cpsievert** [ropensci/plotly#983]
Works ok

```r
p <- plot_ly(txhousing, x = ~month, y = ~log(sales), frame = ~year, split = ~city, color = I("red"))
add_lines(p) 
```
![ani-broke](https://cloud.githubusercontent.com/assets/1365941/25509805/262b0b90-2b81-11e7-9a5a-9ea1b24d3e1d.gif)

doesn't:

```r
p <- plot_ly(txhousing, x = ~month, y = ~log(sales), frame = ~year, split = ~city)
add_lines(p) 
```

![ani-broke2](https://cloud.githubusercontent.com/assets/1365941/25509849/78a1b950-2b81-11e7-8012-aa29e718124d.gif)

@rreusser I *think* this is a plotly.js issue -- any chance default colors aren't being supplied properly?

**Comment by rreusser** [comment]
Hmm… interesting. Is it the number of traces or do some traces not have any data points when the plot initially loads? Maybe it's taking for granted that the traces exist and are colored.

**Comment by cpsievert** [comment]
@rreusser Both plots have the same number of traces (~40) on initial load. The only difference is that `line.color` is specified for each trace (as well as frame traces) in the first plot (`line.color` isn't specified at all in the second)

**Comment by rreusser** [comment]
Yeah… it seems to be thinking the traces are blank and not supplying the line color default, though I'm not sure why. Is it possible to distill this to a pure json example so it can be debugged directly?

(One small additional note: `line: {simplify: false}` is probably necessary if you're not already using it here.)

**Comment by cpsievert** [comment]
The JSON seems to be too big for codepen? Here is a pastebin -- https://pastebin.com/PthUyEbb

**Comment by rreusser** [comment]
I still need to drop props one by one to locate the problem, but if I prune it down to just `"x"` and `"y"` per frame, it seems to work: https://gist.github.com/41d7ba012ccbcb0d60617affbb60f090

Also, for the sake of quick testing, you can run:

```bash
$ npm install -g plotly-mock-viewer
$ curl https://gist.githubusercontent.com/rreusser/41d7ba012ccbcb0d60617affbb60f090/raw/0a7b509ad5dc49f4277f4ff111daf4e9b0e588da/omg-the-vcs-across-the-table-are-so-profoundly-irritating.json -o sample.json
$ plotly-mock-viewer sample.json
```

(doesn't yet load remote urls automatically)
